pkg/database/openshiftclusters.go

```diff
@@ -16,15 +17,17 @@ import (
 )
 
 type openShiftClusters struct {
-	c    cosmosdb.OpenShiftClusterDocumentClient
-	uuid string
+	c     cosmosdb.OpenShiftClusterDocumentClient
+	collc cosmosdb.CollectionClient
+	uuid  string
 }
 
 // OpenShiftClusters is the database interface for OpenShiftClusterDocuments
 type OpenShiftClusters interface {
 	Create(context.Context, *api.OpenShiftClusterDocument) (*api.OpenShiftClusterDocument, error)
 	ListAll(context.Context) (*api.OpenShiftClusterDocuments, error)
 	Get(context.Context, string) (*api.OpenShiftClusterDocument, error)
+	QueueLength(context.Context) (int, error)
```

Comment:
Will add unit test for this as separate PR. Already in progress. 
This will start mocking these interfaces so we can expand